src/utils/terminal_snippet.cr

```diff
@@ -118,13 +118,13 @@ module Mint
         "#{filename}:#{line}:#{column}"
 
       divider =
-        ("─" * (min_width - title.size - gutter_width - 5)).colorize.mode(:dim)
+        ("─" * (min_width - title.size - gutter_width - 5).clamp(0, Int32::MAX)).colorize.mode(:dim)
 
       gutter_divider =
         "─" * gutter_width
 
       footer_divider =
-        "─" * (min_width - gutter_width - 3)
+        "─" * (min_width - gutter_width - 3).clamp(0, Int32::MAX)
```

Comment:
```suggestion
        "─" * (min_width - gutter_width - 3).clamp(0, nil)
```

Reply:
Why? :)

Reply:
You can use an open-ended range instead of passing `MAX` value - it will raise on overflow anyway...